VizNetworkApp/ViewController.swift

```diff
@@ -71,6 +74,25 @@ class ViewController: UIViewController, UITableViewDataSource {
         }
         
     }
+
+    
+    
+    /* Delta time is about 26-27 seconds */
```

Comment:
This is good info. Apparently all BG tasks are terminated after  **about 26-27 seconds**